src/utils.js

```diff
@@ -38,7 +38,11 @@ export function standardizeWitUnits(amount, unit) {
     [`${WIT_UNIT.MICRO}`]: 3,
     [`${WIT_UNIT.NANO}`]: 0,
   }
-  return (amount / Math.pow(10, units[unit])).toFixed(units[unit]).replace(/\.?0+$/, '')
+  if (unit === WIT_UNIT.NANO) {
+    return (amount / Math.pow(10, units[unit])).toFixed(units[unit])
```

Comment:
Hey I think the division in line 42 is redundant as `units[WIT_UNIT.NANO]` always equals `0` and `Math.pow(10, 0)` is `1`.

Same happens with `.toFixed(units[unit])`, it does nothing more than `.toFixed()` with no arguments, or simply `.toString()` :wink: